recipes/opentelemetry-cpp/all/conanfile.py

```diff
@@ -44,7 +44,7 @@ def configure(self):
         if self.options.shared:
             try:
                 del self.options.fPIC
-            except Exception:
+            except AttributeError:
```

Comment:
There is a more general fix for this coming with 1.53 once it's deployed with #13753 

```suggestion
            except Exception:
```